chain/network/src/types.rs

```diff
@@ -35,6 +35,10 @@ use std::sync::RwLock;
 
 /// Current latest version of the protocol
 pub const PROTOCOL_VERSION: u32 = 4;
+/// Number of hops a message is allowed to travel before being dropped.
+/// This is used to avoid infinite loop because of inconsistent view of the network
+/// by different nodes.
+pub const ROUTED_MESSAGE_TTL: u8 = 100;
```

Comment:
Do you think that we will ever need to change this? If so then it might be a good idea to put it in `GenesisConfig`

Reply:
In this case 100 is a very large number, so I don't expect we need to change it. This will prevent very odd situations where routing tables are not synced and message are traveling in cycles. I don't expect that there is a legit route with more than 100 peers, and if that is the case, definitely this won't be the actual problem to solve.

Reply:
Added to network config, but didn't expose it in GenesisConfig (or any config file). Right now it is only tweaked in tests.